lib/tags/injected_app_bar.dart

```diff
@@ -105,6 +105,9 @@ class PBAppBarGenerator extends PBGenerator {
 
   @override
   String generate(PBIntermediateNode source, PBContext generatorContext) {
```

Comment:
Rather than returning `null` here and in the Tab Bar, I think we should probably take off the `InjectedAppBar` and `InjectedTabBar` from the `pb_plugin_list_helper.dart` for the time being. 

What do you think?

Reply:
I think that is correct, I will do the corresponding changes. Thanks!